package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "borsh",
+  "version": "0.1.0",
+  "description": "Binary Object Representation Serializer for Hashing",
+  "main": "borsh-js/index.js",
+  "scripts": {
+    "build": "tsc -p ./tsconfig.json",
+    "test": "jest test --runInBand",
+    "dev": "yarn build -w"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nearprotocol/borsh.git"
+  },
+  "keywords": [
+    "serializer",
+    "binary",
+    "serializer",
+    "deserializer",
+    "consistency",
+    "determenistic"
+  ],
+  "author": "Near Inc",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/nearprotocol/borsh/issues"
+  },
+  "homepage": "https://github.com/nearprotocol/borsh#readme",
+  "devDependencies": {
+    "@types/babel__core": "^7.1.2",
```

Comment:
We also need to publish types either in the package itself or in `@types/` (see https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html)  as e.g. `nearlib` doesn't actually provide any types to its users as of now.

Reply:
Hm, didn't fully understand if we need to send PR to https://github.com/DefinitelyTyped/DefinitelyTyped or will it auto-publish from NPM?

Reply:
@ilblackdragon https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package this part about including in our package should be enough

> Hm, didn't fully understand if we need to send PR to https://github.com/DefinitelyTyped/DefinitelyTyped or will it auto-publish from NPM?

PR is needed. if `types` not specified in `package.json`

Reply:
It is _much_ better to supply the types in the package itself

I _hate_ having to do `npm i packageName @types/packageName`

`@types` is for packages that don't (or won't) supply their own

Reply:
hm, I added `types` in `package.json` so I'm assuming that's enough.